src/helpers/messages.js

```diff
@@ -60,7 +60,7 @@ function getHeader(scope) {
     } else if (scope === 'part') {
         header = __('You are about to submit this test part.');
     }
-    return messageHeaderTpl({header});
+    return messageHeaderTpl({ header: header.trim() });
```

Comment:
BTW, have you tried inserting a zero-width space into the translations instead of a regular space, and the trimming logic here? :)

Reply:
Unfortunately, 0 width translation means no translation, so that the original text is used instead. Hence this workaround to trim emptied translations

Reply:
Yeah, I got it, but a zero-width space is still a symbol ;)